### PR TITLE
Add companion plate to the sample OME-TIFF page

### PR DESCRIPTION
**Pull request by sbesson**
Documents the new multi-file OME-TIFF sample

### CODE REVIEW

File: docs/sphinx/ome-tiff/data.rst [hunk 1]
```diff
@@ -236,7 +236,17 @@ resource.
...
-This section lists various examples of OME-TIFF datasets distributed across multiple TIFF files. Both datasets contain a set of 18 × 24 pixel images with black and white text on each plane giving its time, z-depth and channel. Each of the five focal planes is saved as a separate OME-TIFF named :file:`multifile-Zxx.ome.tiff` where `xx` is the index of the focal plane.
+This section lists various examples of OME-TIFF datasets distributed across
+multiple TIFF files.
+
+The first two datasets contain a set of 18 × 24 pixel images with black and
+white text on each plane giving its time, z-depth and channel. Each of the
+five focal planes is saved as a separate OME-TIFF named
+:file:`multifile-Zxx.ome.tiff` where `xx` is the index of the focal plane.
+
+The third datasets contain a plate with 4 wells at position A2, B1, B3 and C2.
```
Comment (mtbc) [inline docs/sphinx/ome-tiff/data.rst:247]:
```suggestion
The third dataset contains a plate with 4 wells at position A2, B1, B3 and C2.
```

File: docs/sphinx/ome-tiff/data.rst [hunk 2]
```diff
@@ -236,7 +236,17 @@ resource.
...
+The first three wells contain one field of view and the fourth well contains
+2 fields of views. Each well sampole is saved as a separate OME-TIFF.
```
Comment (mtbc) [inline docs/sphinx/ome-tiff/data.rst:249]:
```suggestion
2 fields of view. Each well sampole is saved as a separate OME-TIFF.
```

**Comment by dgault** [comment]
With the updated commits this reads well to me. Looks good to merge.

**Comment by joshmoore** [comment]
Merging, but I'm unclear on the rollout plan. I assume it means rolling out 6.1.2 and unpacking it under downloads?

**Pull request merged**